cfme/tests/cloud/test_provisioning.py

```diff
@@ -154,7 +159,17 @@ def test_provision_from_template(appliance, provider, testing_instance, soft_ass
              num_sec=1000,
              delay=60,
              handle_exception=True)
-    soft_assert(instance.does_vm_exist_on_provider(), "Instance wasn't provisioned")
+    return instance
+
+
+@pytest.mark.parametrize('testing_instance', [True, False], ids=["Auto", "Manual"], indirect=True)
+def test_provision_from_template(provider, provisioned_instance, soft_assert):
+    """ Tests instance provision from template
+
+    Metadata:
+        test_flag: provision
+    """
+    soft_assert(provisioned_instance.does_vm_exist_on_provider(), "Instance wasn't provisioned")
```

Comment:
OPTIONAL: it doesn't make sense to use soft_assert here because it is necessary only in case when you don't want to stop test even if this intermediate condition is False.
I'd replace it with regular assert